recipe/meta.yaml

```diff
@@ -1,46 +1,49 @@
-{% set version = "20160523b" %}
+{% set version = "20170520" %}
 
 package:
   name: texlive-core
   version: {{ version }}
 
 source:
   fn: texlive-{{ version }}-source.tar.xz
-  url: http://mirrors.ctan.org/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
-  sha256: a8b32ca47f0a403661a09e202f4567a995beb718c18d8f81ca6d76daa1da21ed
+  url: https://www.ctan.org/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
+  sha256: c0a032aa9fa29e7cb521659779d6415f3b4b062a6a44193fb4e79cfefe7b8954
 
 build:
-  number: 3
+  number: 0
   skip: True  # [win]
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
     - pkg-config
     - cairo 1.14.*
-    - gmp
+    - gmp >=5.0.1,<7
     - icu 58.*
-    - mpfr
+    - mpfr 3.*
```

Comment:
This is now [`mpfr 3.1.*`]( https://github.com/conda-forge/conda-forge.github.io/blob/7cecae63d1a62cedd9f4d1fce8b365d47efc6830/scripts/pin_the_slow_way.py#L64 ).